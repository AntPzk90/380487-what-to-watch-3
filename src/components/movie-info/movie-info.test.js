import React from 'react';
import renderer from 'react-test-renderer';
import MovieInfo from './movie-info.jsx';
import {Provider} from "react-redux";
import configureStore from 'redux-mock-store';

const mockStore = configureStore([]);

const showFilmCardMock = {
  title: `Bohemian rhapsody`,
  src: `bohemian-rhapsody.jpg`,
  poster: `bohemian-rhapsody.jpg`,
  titlePoster: `bohemian-rhapsody.jpg`,
  genre: `Comedy`,
  releaseDate: `2015`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

it(`SnapshotTest MovieInfo`, () => {

  const store = mockStore({
    showFilmCard: showFilmCardMock,
    activePage: `overview`,
  });

  const tree = renderer
    .create(
        <Provider store={store}>
          <MovieInfo
            film = {store.showFilmCard}
            activePage = {store.activePage}
            onTabClick={() => {}}
          />
        </Provider>, {
          createNodeMock: () => {
            return {};
          }
        })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
