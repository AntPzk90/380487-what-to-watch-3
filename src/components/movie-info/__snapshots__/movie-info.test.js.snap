// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest Movie-info 1`] = `
<div>
  Loading...
</div>
`;
